src/components/shared/Draggable.js

```diff
@@ -67,7 +67,8 @@ export class Draggable extends React.PureComponent<Props, State> {
         e.pageY - mouseDownY,
         true
       );
-      e.stopPropagation();
+      // Note: no stopPropagation, so that other handlers (eg: screenshot
+      // hovers) can also get the event and handle it.
```

Comment:
Thanks for the comment. I tested on other draggable areas to see if everything is ok, and looks good to me.